apps/consumer-client/pages/pcdpass-examples/zkeddsa-proof.tsx

```diff
@@ -0,0 +1,287 @@
+import { EdDSATicketPCDPackage } from "@pcd/eddsa-ticket-pcd";
+import {
+  constructPassportPcdGetRequestUrl,
+  openPassportPopup,
+  usePassportPopupMessages,
+  useSerializedPCD
+} from "@pcd/passport-interface";
+import { ArgumentTypeName } from "@pcd/pcd-types";
+import { generateMessageHash } from "@pcd/semaphore-group-pcd";
```

Comment:
i'm not sure this is a good reason to depend on `@pcd/semaphore-group-pcd"`, i think there's room for some sort of utility package, or even copying this function into this pcd is slightly preferable to having to depend on `@pcd/semaphore-group-pcd`